mev-commit-cli.sh

```diff
@@ -65,7 +65,12 @@ NEXT_PUBLIC_WALLET_CONNECT_ID=
 DD_KEY=${datadog_key}
 EOF
 
-    make -C "$GETH_POA_PATH"/geth-poa up-prod-settlement
+    # Run settlement layer
+    # Set environment variables
+    export AGENT_BASE_IMAGE=nil
+    export L2_NODE_URL=nil
+    # Run Docker Compose
+    docker compose --profile settlement -f "$GETH_POA_PATH/geth-poa/docker-compose.yml" up -d --build
```

Comment:
We should deploy bridge components as well, pattern matching `make up-prod-build`